src/parser/mod.rs

```diff
@@ -7295,6 +7296,26 @@ impl<'a> Parser<'a> {
         }
     }
 
+    /// Parse a numeric literal and return Expr::UnaryOp if the number is signed,
+    /// otherwise return Expr::Value
```

Comment:
```suggestion
    /// Parse a numeric literal as an expression. Returns a [`Expr::UnaryOp`] if the number is signed,
    /// otherwise returns a [`Expr::Value`]
```